### ISSUE TITLE
Manually Building

### ISSUE THREAD

**Issue by Proxy3200** [Nick-Vinesmoke/The-Murk-stealer#78]
Hi Sir, sorry for bothering you again. Today I was just testing Manual Building, so that I can Have my Custom icon and custom File Name and Description. I changed that assembly info(application and description) from "Secure" Folder with **version.py**. Then after that I inserted the Webhook in TheMurk.py as you mentioned, After that When i went to "builder" folder and clicked on Compile.py in the builder folder. It made 2 new folder and but in the folder "TheMurkbuilder.dist" There was no exe file. After that i go to "The Murk" folder, Changed the two values of file grabber and antivirus killer as True. and clicked CompilerPyinstaller.py. I finally formed the .exe file in the "dist" folder. But when I run it nothing happened. Can you please tell what may have gone wrong.

**Comment by Nick-Vinesmoke** [comment]
what you wrote here sounds like nonsense, can you explain more precisely or add me to the dicord Nick Vinesmoke#5304

**Comment by Proxy3200** [comment]
Sorry, I think you just updated read me. I just want to ask can i change these values to true. Does it affect the stub file ?

enableFileGrubber = False #enable file grabber
oneStart = False #enable this if you want logs to come only from unique computers
avKiller = False #enable antiviruses killer

Also, Can i change the icon.ico to other custom icon and these lines in secure folder ?

kids=[
    StringFileInfo(
      [
      StringTable(
        '000004b0',
        [StringStruct('Comments', ' Hail to Thee in the Victor\'s Crown'),
        StringStruct('LegalCopyright', 'Copyright \xA9 Otto von Bismarck  1871'),
        StringStruct('CompanyName', 'German Empire'),
        StringStruct('FileDescription', 'God with us'),
        StringStruct('FileVersion', '8.1.0.0'),
        StringStruct('ProductVersion', '8.1.0.0'),
        StringStruct('InternalName', 'TheMurk.exe'),
        StringStruct('LegalTrademarks', ''),
        StringStruct('OriginalFilename', 'TheMurk.exe'),
        StringStruct('ProductName', 'German Empire'),
        StringStruct('BuildID', '070523803')])

Basically, i just wanted to have custom icon and custom assembly info so i changed the inside values of StringStruct. I don't know if it affect the murk stealer but when i finished building with nuitikacompiler.py and run the stub. I didnt recieved any logs on my telegram. I checked the chat id and http api both are correct. I don't know what has gone wrong.

**Comment by Nick-Vinesmoke** [comment]
This error may be due to an incorrect change in the file version in the properties or an incorrect indication of the telegram or send type

**Comment by Proxy3200** [comment]
Pardon Sir, But I have not changed the virus version. I only changed the Internal name, Original file name, Product Name and the file description. The telegram http api and chat if are all correct when i build the with builder they work correctly but the problem with builder is I cannot modify the above info in it.

**Comment by Nick-Vinesmoke** [comment]
Idk mb you build manually not correctly, but you can use Resource Hacker to change builded file
